src/api.jl

```diff
@@ -221,10 +233,23 @@ function _minimal_period(ds::DiscreteTimeDynamicalSystem, po::PeriodicOrbit, ato
     return po
 end
 
-function _minimal_period(ds::ContinuousTimeDynamicalSystem, po::PeriodicOrbit, atol)
-    # TODO
-    minT_po = po
-    return minT_po
+function _minimal_period(ds::ContinuousTimeDynamicalSystem, po::PeriodicOrbit, atol; maxiter=10, stepsize=1e-4)
```

Comment:
why is `atol` an argument and the rest are keyword arguments? I think all three should be keywords.